### PR TITLE
Unify POS app card markup

### PR DESCRIPTION
**Pull request by bolatovumar**
See discussion here: https://github.com/btcpayserver/btcpayserver/discussions/2431, particularly this comment: https://github.com/btcpayserver/btcpayserver/discussions/2431#discussioncomment-587869

**Before (notice the button looks different):**

Screen Shot 2021-04-10 at 6 33 42 PM

**After:**

Screen Shot 2021-04-10 at 6 33 22 PM

### CODE REVIEW

File: BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml [hunk 1]
```diff
@@ -120,3 +90,31 @@
         </div>
     </div>
 </div>
+
+@functions {
+    private void PayFormInputContent(string buttonText, decimal? minPriceValue = null, decimal? priceValue = null)
```
Comment (bolatovumar) [inline BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml:95]: Split shared markup which was almost the same but slightly different previously into these two functions to ensure the mark-up is the same.

**Comment by MaiHistoryCompanyLimited** [comment]
{#2453}

**Review by dennisreimann — APPROVED** [review]

File: BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml (hunk 1 above)
Comment (Kukks) [reply BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml:95]: Interesting take, you've recreated React in .net razor

Comment (dennisreimann) [reply BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml:95]: Is there a way to define inline view components? That might be a good alternative here …

Comment (bolatovumar) [reply BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml:95]: I couldn't find a way to do it inline. There are a few ways to do it not in line while still componentizing the mark-up and this one seemed to work best.

Comment (NicolasDorier) [reply BTCPayServer/Views/AppsPublic/PointOfSale/Static.cshtml:95]: cool I did not know we could do that in razor.

**Review by NicolasDorier — APPROVED** [review]

**Pull request merged**